packages/launchpad/cypress/e2e/config-warning.cy.ts

```diff
@@ -91,7 +111,7 @@ describe('experimentalSingleTabRunMode', () => {
 })
 
 describe('experimentalStudio', () => {
-  it('is not a valid config for component testing', () => {
+  it('is not a valid config for component testing', { defaultCommandTimeout: THIRTY_SECONDS }, () => {
```

Comment:
Are we bumping the default command timeout for all of the commands because `cy. findByTestId` doesn't have a custom timeout option?

Reply:
Looks that way: https://testing-library.com/docs/queries/bytestid/#options

Reply:
Whatttttt this is so cool! I didn't realize you could pass in test config there!